src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue18443.cs

```diff
@@ -0,0 +1,23 @@
+﻿#if !MACCATALYST && !IOS
+using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+namespace Microsoft.Maui.TestCases.Tests.Issues
+{
+	public class Issue18443 : _IssuesUITest
+	{
+		public Issue18443(TestDevice device) : base(device) { }
+
+		public override string Issue => "[Android] SelectionLength Property Not Applied to Entry at Runtime";
```

Comment:
Its kind of dumb, but I think this string needs to match exactly what is in the `[Issue]` attribute in the UI (so `Issue18443.xaml.cs`) else it won't navigate to the test correctly

Reply:
I have modified the code accordingly.